obspy/io/sac/util.py

```diff
@@ -171,7 +171,7 @@ def _clean_str(value, strip_whitespace=True):
     Trace that the user may have manually added.
     """
     try:
-        value = value.decode('ASCII', 'replace')
+        value = value.decode(encode, 'replace')
```

Comment:
I think it would be a good idea to make this something like..

```python
try:
    value = value.decode(encoding, 'strict')
except UnicodeError:
    msg = ('Encountered characters that can not be '
           'decoded with encoding {}: {}\nThese characters '
           'are ignored.'.format(encoding, value))
    warnings.warn(msg)
    value = value.decode(encoding, 'replace')
except AttributeError:
    ...
```